packages/firebase_auth/firebase_auth/example/ios/Runner/main.m

```diff
@@ -2,7 +2,7 @@
 #import <UIKit/UIKit.h>
 #import "AppDelegate.h"
 
-int main(int argc, char* argv[]) {
+int main(int argc, char *argv[]) {
```

Comment:
maybe switch this back. All other example `main()` functions have the previous style, keeps it a wee bit consistent 😄

Reply:
Let's go, I guess clang-format wanted it the other way 🤔